packages/assert/src/assert.js

```diff
@@ -1,9 +1,10 @@
 // Copyright (C) 2019 Agoric, under Apache License 2.0
 
-/* global harden */
-
 // @ts-check
 
+// eslint-disable-next-line spaced-comment
+/// <reference types="ses"/>
```

Comment:
What does this do?

Reply:
... and is the triple slash intentional?

Reply:
CF: https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-

This imports the types from the `ses` package, namely global `harden` with the correct, type-preserving return value.